Sources/Plasma/PubUtilLib/plNetCommon/plNetCommonHelpers.cpp

```diff
@@ -242,7 +242,7 @@ void plCreatableListHelper::Read( hsStream* s, hsResMgr* mgr )
         s->LogSubStreamPushDesc("Compressed Data");
         s->Read( zBufSz, (void*)zBuf.data() );
         plZlibCompress compressor;
-        uint32_t tmp;
+        uint32_t tmp = bufSz;
```

Comment:
```suggestion
        uint32_t tmp = 0;
```

I'm not sure I understand what's going on, but I don't think `tmp` needs to default to what the assertion expects.